api/src/backend/api/specs/v1.yaml

```
@@ -8918,6 +8918,19 @@ components:
                     default: output
                 required:
                 - bucket_name
+              - type: object
+                title: AWS Security Hub
+                properties:
+                  send_only_fails:
+                    type: boolean
+                    default: false
+                    description: If true, only findings with status 'FAILED' will
```

```suggestion
                    description: If true, only findings with status 'FAIL' will
```

`FAILED` is how AWS Security Hub names failed findings.

---

Please, change it in other places.

---

Done